README.md

```diff
@@ -91,8 +91,8 @@ apt-get update
 ###### Uninstall
 
 ```sh
-rm /etc/yum.repos.d/balena-etcher.repo
-rm /etc/yum.repos.d/balena-etcher-source.repo
+sudo rm /etc/yum.repos.d/balena-etcher.repo
```

Comment:
Presumably this section is also missing a command to actually uninstall the `balena-etcher-electron` package?

Reply:
Not sure about yum stuff...but for ubuntu everything works afaik